docs/modules/ROOT/examples/tutorial/backup.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   failedJobsHistoryLimit: 2
   successfulJobsHistoryLimit: 2
-  promURL: http://minio:9000
+#  promURL: http://minio:9000
```

Comment:
Nit: We could probably remove this completely from the samples? It doesn't work anyway out of the box without a prometheus pushgw.